.builders/build.py

```diff
@@ -153,6 +154,10 @@ def build_macos():
         final_requirements = mount_dir / 'frozen.txt'
         shutil.move(final_requirements, output_dir)
 
+        # Move the dependency sizes to the output directory
+        dependency_sizes_dir = mount_dir / 'sizes.json'
```

Comment:
this looks like a duplicated code. I would then create a function and reuse it in both `build_image` and `build_macos`

Reply:
Same as for my previous comment above. All of this section is mostly duplicated, including the copy of wheels, requirements and the creation of the directories. Maybe this can be done in a follow up PR as an overall improvement?